### PR TITLE
Search page using Algolia

### PR DESCRIPTION
**Pull request by laurentlb**
Preview: https://i.imgur.com/RJkHY2R.png

Since it's not fully integrated, I'm adding a new page (instead of
replacing search.html). It also allows us to test it live before
doing the switch.

### CODE REVIEW

File: search2.html [hunk 1]
```diff
@@ -0,0 +1,71 @@
+---
+layout: default
+title: Search
+---
+
+<link rel="stylesheet" type="text/css" href="https://cdn.jsdelivr.net/npm/instantsearch.js@2.10.4/dist/instantsearch.min.css">
```
Comment (jin) [inline search2.html]: let's vendor these into the repo?

Comment (laurentlb) [reply search2.html]: Done, thanks!

File: search2.html [hunk 2]
```diff
@@ -0,0 +1,70 @@
+---
+layout: default
+title: Search
+---
+
+<link rel="stylesheet" type="text/css" href="/assets/css/instantsearch.min.css">
+<link rel="stylesheet" type="text/css" href="/assets/css/instantsearch-theme-algolia.min.css">
+
+<script src="https://cdn.jsdelivr.net/npm/instantsearch.js@2.10.4"></script>
```
Comment (jin) [inline search2.html]: oops, missed this. also vendor javascripts?

Comment (laurentlb) [reply search2.html]: Done

**Pull request merged**